db/seeds.rb

```diff
@@ -152,3 +202,15 @@
     phone: '012-345-6789', reservations: r_and_d.sample, delivery: r_and_d.sample
     )
 end
+
+user = User.new(email: "seed@seeder.com", password: "seedpassword")
+user.save!
+
+Restaurant.all.each do |restaurant|
+  10.times do
+    restaurant.reviews.create(
```

Comment:
utilize `create!` here to ensure the review adheres to validation. Ideally you should only create reviews if the count is 0 for a given restaurant